release/sil/sil.brb.brao/sil.brb.brao.model_info

```diff
@@ -0,0 +1,7 @@
+{
+    "license": "mit",
+    "languages": [
+        "brb"
```

Comment:
Maybe also include `brb-khmr` (here and .model.kps) so this model can be used with sil_khmer
https://keyman-staging.com/keyboards/sil_khmer?bcp47=brb-khmr#expand-languages